vast/vast2/wrapper.go

```diff
@@ -1,16 +1,18 @@
-package vast
+package vast2
+
+import vastbasic "github.com/Vungle/vungo/vast/basic"
 
 // Wrapper type represents a <Wrapper> element in VAST document that contains a single URI
 // reference to a vendor ad server, a.k.a. third party ad server, where the actual VAST w
 // content will be served. The vendor ad server may also provide additional wrapper which
 // eventually resolves to the actual ad.
 type Wrapper struct {
-	AdSystem     *AdSystem          `xml:"AdSystem"`     // Required.
-	VastAdTagURI string             `xml:"VASTAdTagURI"` // Required.
-	Impressions  []*Impression      `xml:"Impression"`   // Required.
-	Errors       []string           `xml:"Error,omitempty"`
-	Creatives    []*CreativeWrapper `xml:"Creatives>Creative"` // Required.
-	Extensions   []*Extension       `xml:"Extensions>Extension,omitempty"`
+	AdSystem     *vastbasic.AdSystem     `xml:"AdSystem"`     // Required.
+	VastAdTagURI string                  `xml:"VASTAdTagURI"` // Required.
```

Comment:
should defined it as trimdata

Reply:
Change to TrimmedData to enable the ability to trim space for URI.